corehq/apps/app_manager/views/modules.py

```diff
@@ -882,11 +881,7 @@ def overwrite_module_case_list(request, domain, app_id, module_unique_id):
             ", ".join(map(str, not_updated_modules)))
         messages.error(request, _error_msg)
 
-    if updated_modules:
-        app.save()  # Save successfully overwritten menus.
-        _msg = _('Case list configuration updated from {} menu to {} menu(s).').format(
-            src_module.default_name(), ", ".join(map(str, updated_modules)))
-        messages.success(request, _msg)
+    app.save()
```

Comment:
Was not the list useful? In case an error in all modules requesting updates, app save would not be necessary but would happen now?

In fact, the save should happen if there are no errors?

Reply:
nit: that consolidated message seemed like a nice summary of what actually happened, sharing the "functional change", though users might be just ignoring it and considering something green to be good. Though shorter messages are nice.

Now they see,
```
Updated Registration
Updated Follow Up
```

Fine to punt but was thinking if we could add something on top to have similar update
```
Updating Base Module
Updated Registration
Updated Follow up
```

Reply:
I had mixed feelings about doing this. It does mean that the user will see multiple messages, instead of a single success and/or single error message. But I really wanted to simplify this code, to reduce the local data and the branching, because we've had a couple of bugs in this area and the real bug in USH-960 is still outstanding.